### PR TITLE
Update example/ios to match new plugin system

### PR DESCRIPTION
**Pull request by mit-mit**
As per breaking change https://groups.google.com/forum/#!msg/flutter-dev/zba1Ynf2OKM/6m__2uCbAgAJ

### CODE REVIEW

File: example/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -161,6 +161,7 @@
 				TargetAttributes = {
 					97C146ED1CF9000F007C117D = {
 						CreatedOnToolsVersion = 7.3.1;
+						DevelopmentTeam = EQHXZ8M8AV;
```
Comment (mravn-google) [inline example/ios/Runner.xcodeproj/project.pbxproj]: Should not be added (three times)

File: example/ios/Runner/GeneratedPluginRegistrant.h [hunk 2]
```diff
@@ -7,7 +7,6 @@
 
 #import <Flutter/Flutter.h>
 
-
```
Comment (mravn-google) [inline example/ios/Runner/GeneratedPluginRegistrant.h]: Unnecessary change.

File: example/ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (mit-mit) [inline example/ios/Runner.xcodeproj/project.pbxproj]: Done

File: example/ios/Runner/GeneratedPluginRegistrant.h (hunk 2 above)
Comment (mit-mit) [inline example/ios/Runner/GeneratedPluginRegistrant.h]: Reverted

**Comment by mit-mit** [comment]
cc @lukef
